benchmarks/time.rs

```diff
@@ -207,5 +209,52 @@ setup_benchmark! {
         ben.iter(|| time!(12:00) > time!(11:00));
         ben.iter(|| Time::MIDNIGHT == time!(0:00:00.000_000_001));
     }
+
+    fn sort(ben: &mut Bencher<'_>) {
+        ben.iter_batched_ref(
+            || {
+                let mut res = vec![];
+                let mut last = Time::MIDNIGHT;
+                for _ in 0..4096 {
+                    res.push(last);
+                    last += 997.std_milliseconds();
+                }
+                res.sort_unstable_by_key(|t|
+                    (t.nanosecond(),t.second(),t.minute(),t.hour())
+                );
+                res
+            },
+            |v| black_box(v).sort_unstable(),
+            criterion::BatchSize::SmallInput
+        )
+    }
+
+    fn sort_unaligned(ben: &mut Bencher<'_>) {
```

Comment:
What is the difference between this and `sort`, in terms of what's being benchmarked?

Reply:
Gotcha. I'll take a look at the PR soon. Looks like CI will pass, so that's not a concern.